data/ships.txt

```diff
@@ -2136,6 +2176,60 @@ ship "Osprey"
 
 
 
+ship "Prospector"
```

Comment:
Only so many words in the english language, man.  I'll look for another name again just to be nice ;)